src/map/battle.c

```diff
@@ -5271,9 +5271,9 @@ struct Damage battle_calc_weapon_attack(struct block_list *src,struct block_list
 				case RL_B_FLICKER_ATK:
 					rskill = RL_FLICKER;
 					break;
-				case RL_GLITTERING_GREED_ATK:
-					rskill = RL_GLITTERING_GREED;
-					break;
+				// case RL_GLITTERING_GREED_ATK:
+				//	rskill = RL_GLITTERING_GREED;
+				//	break;
 				default:
```

Comment:
Is this intended?

Reply:
Should just remove it altogether as the skill doesn't exist.